From 38800b2b4d9affc1d068a929e57cbb70dc824e37 Mon Sep 17 00:00:00 2001
From: Matthieu Gallien <matthieu_gallien@yahoo.fr>
Date: Wed, 27 May 2020 12:27:01 +0200
Subject: [PATCH] PlaceholderMessage depends on Qt 5.13 even if CMakeLists.txt
 requires 5.12

the import of QtQuick.Layouts in PlaceholderMessage.qml requires 1.13 in
contradiction with the required Qt version in CMakeLists.txt
---
 src/controls/PlaceholderMessage.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/controls/PlaceholderMessage.qml b/src/controls/PlaceholderMessage.qml
index 574fbf10..681d9f0e 100644
--- a/src/controls/PlaceholderMessage.qml
+++ b/src/controls/PlaceholderMessage.qml
@@ -6,7 +6,7 @@
 
 import QtQuick 2.0
 import QtQuick.Controls 2.4 as QQC2
-import QtQuick.Layouts 1.13
+import QtQuick.Layouts 1.12
 import org.kde.kirigami 2.12 as Kirigami
 
 import "private"
-- 
2.26.2

